content/en/docs/components/pipelines/v1/installation/choose-executor.md

```diff
@@ -10,68 +10,21 @@ Kubeflow Pipelines runs on [Argo Workflows](https://argoproj.github.io/workflows
-
-{{% alert title="Warning" color="warning" %}}
-Docker executor depends on docker container runtime, which will be deprecated on Kubernetes 1.20+.
-{{% /alert %}}
-
-* Container Runtime: docker only. However, [Kubernetes is deprecating Docker as a container runtime after v1.20](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/).
-  On Google Kubernetes Engine (GKE) 1.19+, container runtime already defaults to containerd.
-* Reliability: most well-tested and most popular argo workflows executor
-* Security: least secure
-  * It requires `privileged` access to `docker.sock` of the host to be mounted which.
-  Often rejected by Open Policy Agent (OPA) or your Pod Security Policy (PSP).
-  GKE autopilot mode also rejects it, because [No privileged Pods](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview#no_privileged_pods).
-  * It can escape the privileges of the pod's service account.
-
-#### Prepare a GKE cluster for Docker Executor
-
-For GKE, the node image decides which container runtime is used. To use docker
-container runtime, you need to [specify a node image](https://cloud.google.com/kubernetes-engine/docs/how-to/node-images) with Docker.
-
-You must use one of the following node images:
-
-* Container-Optimized OS with Docker (cos)
-* Ubuntu with Docker (ubuntu)
-
-If your nodes are not using docker as container runtime, when you run pipelines
-you will always find error messages like:
-
-> This step is in Error state with this message: failed to save outputs: Error response from daemon: No such container: XXXXXX
+team only recommend choosing between emissary executor and docker executor.
 
 ### Emissary Executor
 
-{{% alpha-status feedbacklink="https://github.com/kubeflow/pipelines/issues/6249" %}}
-
-Emissary executor is a new workflow executor. It was first released in Argo Workflows v3.1 (June 2021).
-However, the Kubeflow Pipelines team believe that its architectural and portability
-improvements can make it the default executor that most people should use in the
-future.
-
-Therefore, the team makes a commitment to actively collect feedback and fix bugs
-for the emissary executor, so that we can stablize it faster.
-Submit your feedback in [the Emissary Executor feedback github issue](https://github.com/kubeflow/pipelines/issues/6249).
-
-So far, Kubeflow
-Pipelines test infrastructure has been running stably with the emissary executor.
+Emissary executor is the **default** workflow executor for Kubeflow Pipelines v1.8+. It was first released in Argo Workflows v3.1 (June 2021).
+The Kubeflow Pipelines team believe that its architectural and portability
+improvements can make it the default executor that most people should use going forward.
 
 * Container Runtime: any
 * Reliability: not yet well-tested and not yet popular, but the Kubeflow Pipelines
```

Comment:
We can probably update this sentence to `Well tested and it is the default Argo executor from >= v3.3.`. But feel free to merge the current change without update, since our main goal is to call out the default executor.